src/core/Link/LinkList/LinkList.md

```diff
@@ -0,0 +1,44 @@
+### Link List
+
+A styled list for providing a list of links.
+
+Make sure you use a descriptive heading or a label with the list and link it to the list via the `ariaDescribedBy` prop. Make sure you use the right heading level semantically and use the `as` prop for changing the styling if needed.
+
+```js
+import {
+  Link,
+  ExternalLink,
+  LinkListItem,
+  RouterLink,
+  LinkList,
+  Heading
+} from 'suomifi-ui-components';
+
+<div>
+  <Heading variant="h5" as="h3" id="otsikko">
+    Useful resources
+  </Heading>
+  <LinkList aria-describedby="otsikko">
+    <LinkListItem>
```

Comment:
There could be more instructions about using this component: it takes LinkListItem as children, and those take Link/ExternalLink/RouterLink etc. Now it's shown in this example and not stated in documentation otherwise

Reply:
Added this info to the main level instructions. It'd be clearer if the typing of LinkListItem's children prop would display properly in styleguidist. Added that typing info to the instructions as well. Technically the children of the list itself can be pretty much anything at the moment.